specification/app/resource-manager/Microsoft.App/stable/2025-07-01/examples/ContainerApps_Kind_CreateOrUpdate.json

```diff
@@ -0,0 +1,200 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "containerAppName": "testcontainerAppKind",
+    "api-version": "2025-07-01",
+    "containerAppEnvelope": {
+      "location": "East Us",
+      "managedBy": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.Web/sites/testcontainerAppKind",
+      "kind": "workflowapp",
+      "properties": {
+        "managedEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/testmanagedenv3",
+        "configuration": {
+          "activeRevisionsMode": "Single",
+          "ingress": {
+            "external": true,
+            "targetPort": 80,
+            "allowInsecure": true
```

Comment:
>allowInsecure [](http://example.com/codeflow?start=13&length=13)

consider putting insecure 'false' instead of true in the examples, since 
a) people are likely to copy them, and we'd like to b) push people into the pit of security success